packages/back-end/src/controllers/experiments.ts

```diff
@@ -1179,6 +1179,8 @@ export async function postMetrics(
     tags,
     winRisk,
     loseRisk,
+    varianceThreshold,
+    minThresholdSignificance,
```

Comment:
These don't have anything to do with variance or significance.  I would change the name of these throughout the PR to something like `maxPercentChange` and  `minSampleSize` instead.